lib/fedex_rest_api/address.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "fedex_rest_api/base"
+
+class FedexRestApi::Address
+  attr_reader :address_object
+
+  ADDRESS_VALIDATION_URL = "/address/v1/addresses/resolve"
```

Comment:
I wonder if it would be useful to standardize this constant amongst all classes to something generic like `ENDPOINT_URL` because it might allow us to pull methods and constants out into modules or shared classes/interfaces in the future.  For example, we could have a module that holds the method `env_url` and it would reference the constant.  Not a blocker, but the consistency might be useful...someday.

Reply:
I think that could be useful/something interesting to consider as this continues to get refined. My next steps will be to break out and make 'class objects' for each of the API types so I can keep it in mind as I roll through that. 👍